Lombiq.Walkthroughs/Assets/Scripts/walkthroughs.js

```diff
@@ -3177,8 +3177,28 @@ jQuery(($) => {
                     },
                     {
                         title: 'Roles',
-                        text: 'Let\'s see one! Click on the <em>"Edit"</em> button.',
-                        attachTo: { element: '.edit.btn.btn-primary.btn-sm', on: 'top' },
+                        text: `<p>The System permissions at the top can't be deleted, but Anonymous and Authenticated
+                            can be changed. Administrator, however, can't: it's a special role that has all the
+                            permissions, current and future. Users with this role can do anything! Be careful who you
+                            give this to. By the way, you're one of them :).</p>`,
```

Comment:
The `:).` is pretty confusing at first sight, because you see one cluster of punctuation instead of two symbols. You could just use an emoji to make it less noisy.

```suggestion
                            give this to. By the way, you're one of them 🙂.</p>`,
```

Reply:
Is this a Gen Z problem I'm too Millenial to understand? :D

Reply:
breh, I was born in 1990 :smiling_face_with_tear:

Reply:
sounds like a skill issue

Reply:
Sorry, I will try to spawn earlier next time. [backwards long jumps into a parallel universe]

Reply:
May I interest you in: https://github.com/Lombiq/UI-Testing-Toolbox/blob/15d6abdaedb797926714be6a06efed30cc537af5/Lombiq.Tests.UI.Shortcuts/Services/TimeShiftingClock.cs#L13